src/components/Home.js

```diff
@@ -29,43 +32,53 @@ const Home = ({ handleTokenCreation, setToken, setCollectionId }) => {
   };
 
   return (
-    <main>
-      <button onClick={handleTokenCreation} className="add-button">
-        Create a Shopping List
-      </button>
-
-      <div>
-        <h1>Join an existing shopping list by entering a token</h1>
-      </div>
+    <div className="wrapper">
+      <div className="app">
+        <div className="app__dots-shadow panel">
+          <main className="align-center">
+            <Header />
+            <div>
+              <button
+                onClick={handleTokenCreation}
+                className="app__create-button"
+              >
+                Create a New List
+                <span className="create-button__icon">
+                  <ArrowIcon />
+                </span>
+              </button>
 
-      <form className="form">
-        <div className="form-row">
-          <div className="form-group">
-            <label htmlFor="existing-token">Share Token</label>
-            <input
-              type="text"
-              id="existing-token"
-              className="form-field"
-              placeholder="Enter three word token"
-              value={value}
-              onChange={handleChange}
-            />
-          </div>
+              <form className="app__token-form flow">
+                <label htmlFor="token" className="visually-hidden">
+                  Join existing list with a token
+                </label>
+                <div className="token-form__field">
+                  <input
+                    type="text"
+                    id="token"
+                    className="token-form__input"
+                    placeholder="Join with a token"
+                    value={value}
+                    onChange={handleChange}
+                  />
+                  <button
+                    type="submit"
+                    onClick={joinList}
+                    className="token-form__button"
+                    disabled={value.length < 1}
+                  >
+                    <span className="icon">
+                      <SubmitIcon />
+                    </span>
+                  </button>
+                </div>
+              </form>
+            </div>
```

Comment:
Not a deal-breaker for me at all, but I'd suggest a bit of spacing at the top of that 'token does not exist' message for style purposes. Totally optional though if you want to add it or not 👍

Reply:
Totally agree - I fixed it👍